### DISCUSSION TITLE
Shortcut CTRL+1 didn't switch to standard mode on linux

### DISCUSSION THREAD

**Discussion by NotRealSean** [ppy/osu#25538] [Q&A] [answered]
### Type

Game behaviour

### Bug description

just CTRL+1 wont work on linux

### Screenshots or videos

https://github.com/ppy/osu/assets/89067287/40dc62b4-62bb-4dd8-9482-6cfeb5849ee6

### Version

2023.1114.1-1

### Logs

[database.log](https://github.com/ppy/osu/files/13432843/database.log)
[input.log](https://github.com/ppy/osu/files/13432844/input.log)
[network.log](https://github.com/ppy/osu/files/13432845/network.log)
[performance.log](https://github.com/ppy/osu/files/13432846/performance.log)
[runtime.log](https://github.com/ppy/osu/files/13432847/runtime.log)

**Comment by peppy** [comment, marked as answer]
It looks like you're reporting this to the wrong place. We don't develop linux. You'll need to seek support with your distro/WM instead.
